dependencies/classpath.txt

```diff
@@ -73,6 +73,7 @@ com.squareup.retrofit2:converter-moshi:2.9.0
 com.squareup.retrofit2:retrofit:2.9.0
 com.squareup:javapoet:1.10.0
 com.squareup:javawriter:2.5.0
+com.squareup:kotlinpoet:1.3.0
```

Comment:
Do we know where this was introduced?

Reply:
It's coming from the kotlinx-benchmark plugin. ([build scan](https://scans.gradle.com/s/xkruxa7ysfkh6/build-dependencies?dependencies=kotlinpoet&expandAll&focusedDependency=WzAsMCw0NTQsWzAsMCxbMF1dXQ&focusedDependencyView=versions))

It shouldn't really matter, since this is the build classpath.